doc/source/visualizing/plots.rst

```diff
@@ -905,7 +908,7 @@ customization.
 Colormaps
 ~~~~~~~~~
 
-Each of these functions accept two arguments.  In all cases the first argument
+Each of these functions accepts at least two arguments.  In all cases the first argument
```

Comment:
The idea here was to avoid suggesting these methods have *exactly* two arguments, when it's not a requirement, and leave the details for what other arguments might be to the rest of the text. For instance `PlotContainer.set_log` currently accepts four arguments, including one deprecated.

Do you see a better way to phrase this to avoid set up without payoffs ?

Reply:
No, I see your point.  Sounds good to me!